packages/core/config/utils.js

```diff
@@ -0,0 +1,18 @@
+const {getPaths} = require('@redwoodjs/internal')
```

Comment:
The `core/config` directory contains a bunch of other functionality. I would prefer if this was a function declared inside of `webpack.common.js`

Reply:
Moved to webpack.common.js